tests/sql_repository_test.py

```diff
@@ -0,0 +1,89 @@
+from faker import Faker
+
+fake = Faker()
+
+existing_elements_registry = []
+last_deleted_element = None
+
+
+def test_create(sql_repository):
+    """Should create a new Entry"""
+    global sample_element
+    sample_element = dict(name=fake.name(),
+                          email=fake.safe_email(),
+                          age=fake.pyint(min_value=10, max_value=80))
+
+    result = sql_repository.create(sample_element)
+
+    assert result is not None
+    assert result.id is not None
+    assert result.created_at is not None
+    assert result.created_by is not None
+    assert result.updated_at is None
+    assert result.updated_by is None
+
+    existing_elements_registry.append(result)
+
+
+def test_find(sql_repository):
+    """Should find created element"""
+    existing_element = existing_elements_registry[0]
+
+    found_element = sql_repository.find(existing_element.id)
+
+    assert found_element is not None
+    assert found_element.id is not None
+
+
+def test_update(sql_repository):
+    """Updates an existing element"""
```

Comment:
no need to have comments, please remove them and make test name consistent

Reply:
In Java I would not have put comments, but this is Python. It is common here put comments even in modules. This is a test that tests the function called `update`. This is very simple.